yt/chyt/controller/internal/strawberry/controller.go

```diff
@@ -51,4 +50,7 @@ type Controller interface {
 type ControllerFactory struct {
 	Ctor   func(l log.Logger, ytc yt.Client, root ypath.Path, cluster string, config yson.RawValue) Controller
 	Config yson.RawValue
+	// TODO(max): extra commands is actually of type []api.CmdDescriptor, but we can't import it here
```

Comment:
Does not look so. CmdDescriptor depends on HandlerFunc, which depends on HTTPAPI, which depends on API, which depends on Controller.

It looks like extracting ControllerFactory to a separate module might fix the issue, but it may be better for somebody with broader perspective of the code (e.g you or Dima) to perform such a refactoring. In general, there is a light vibe of overengineering here, especially after my additions of different controller families.